### PR TITLE
WIP: Chicago teachers pension fund #191

### PR DESCRIPTION
**Pull request by cherdeman**
Hey @r-wei  and @bonfirefan! Here's what I've worked on so far on this, though i'm now getting a page does not exist error when I try to find the meeting schedule: https://www.ctpf.org/general_info/boardmeetings.htm. Do you all have the same issue?

### CODE REVIEW

**Comment by bonfirefan** [comment]
Hey @cherdeman, the site is down for me as well. It looks like they may have done a major update of their site, it no longer looks the Geocities/90's. This [appears to be the new page](https://www.ctpf.org/post/board-meetings), though you may be able to find a new resource. It looks like they are keeping the meetings in plaintext despite the website facelift.....

**Comment by r-wei** [comment]
Oh no! That's so unfortunate. @cherdeman , do you want to build out a spider for the new site? If you do, feel free to keep committing changes to this PR. If you don't (which is totally understandable), we'll just close this PR and make an issue that a new spider is needed.

**Comment by cherdeman** [comment]
@r-wei and @bonfirefan I'll work through the site update! I've got a working spider now, though its failing on the validation checks. I must have missed some of the schema updates, anything you notice if you take a look? Also still have some cleanup and need to build out tests.

**Comment by cherdeman** [comment]
Made a few edits to clean up the spider and added more tests. I'm picking up a failure on a different scaper (chi_school_community_action_council) and still failing on the source_url validation.

**Comment by r-wei** [comment]
Hey @cherdeman ,
- The sources url can't be an empty string. We want the url of the site where the information was scraped from. Usually something like `response.url` works.
- I'll fix work on that failing chi_school_community_action_council now

**Comment by r-wei** [comment]
@cherdeman , the chi_school_community_action_council should pass now if you pull the newest changes from master

File: city_scrapers/spiders/chi_teacherpension.py [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+        
+
+    def _parse_classification(self, i):
+        """
+        Parse or generate classification (e.g. public health, education, etc).
+        """
+        if i == 1:
+        	return 'board meeting'
+        else:
+        	return ''
```
Comment (r-wei) [inline city_scrapers/spiders/chi_teacherpension.py]: hey @cherdeman , can you return either "board meeting" or "committee meeting" here?

Comment (cherdeman) [reply city_scrapers/spiders/chi_teacherpension.py]: on line 86? sure thing

**Pull request merged**